internal/app/init/internal/security/cis/cis.go

```diff
@@ -64,20 +63,12 @@ func CreateEncryptionToken() error {
 		return nil
 	}
 
-	random := func(min, max int) int {
-		return rand.Intn(max-min) + min
-	}
-	var encryptionKeySecret string
-	seed := time.Now().Unix()
-	rand.Seed(seed)
```

Comment:
this is a side-effect of this PR, seeding is removed;

looks like `rand` was implicitly seeded in this function, but it's only called for master bootstrap node, going to add global seed at startup for all the services

Reply:
good catch

Reply:
actually looks like there's just one place using `math/rand`, and it's in the unit-test

but it might be somewhere deep in the libraries we're using

Reply:
and in the future who knows if we might need it